### PR TITLE
Label and description updates.

### PR DESCRIPTION
**Pull request by JimmySunDance**
Currently working through the schemas, updating table labels and and descriptions. 

Files will be added in small batches. 

-- NOT READY TO MERGE --

### CODE REVIEW

**Comment by pixie79** [comment]
Bf function types is from the port types of the UNLocode list but with a couple more added specific to our needs,
Events is also cop specific.

**Comment by pixie79** [comment]
Nationalities is an interesting list, it is a merge from Wicu, HOB and foreign office. There is no definitive list of Visa requirements currently published

**Comment by pixie79** [comment]
Marriage another interesting list, I think this was taken from a gds source, so as to keep questions open.

**Comment by JimmySunDance** [comment]
> Looking good. Can we ensure that all entities description have a "period" at the end and the right grammar (commas, etc) if needed.

The full stop is open for discussion, currently there are no full stops on ANY description in any field. Happy to go through and change this.

**Comment by p3dr0migue1** [comment]
> > Looking good. Can we ensure that all entities description have a "period" at the end and the right grammar (commas, etc) if needed.
> 
> The full stop is open for discussion, currently there are no full stops on ANY description in any field. Happy to go through and change this.

Open by whom and where? I know that there aren't any full stops hence the request. Not having the proper grammar means that the front end will display a grammatically incorrect description.

**Comment by JimmySunDance** [comment]
> Open by whom and where? I know that there aren't any full stops hence the request. Not having the proper grammar means that the front end will display a grammatically incorrect description.

Talking with @T0MT33 on slack. He was of the opinion that given there aren't any it didn't matter. Happy to change it though.

**Comment by p3dr0migue1** [comment]
> > Open by whom and where? I know that there aren't any full stops hence the request. Not having the proper grammar means that the front end will display a grammatically incorrect description.
> 
> Talking with @T0MT33 on slack. He was of the opinion that given there aren't any it didn't matter. Happy to change it though.

Right I see. Yes please if you can change it would be great, shouldn't be too hard. This will ensure that everything will be displayed nicely for the users.

File: schemas/reference/V34__tdareasons.sql [hunk 1]
```diff
@@ -6,12 +6,12 @@ CREATE TABLE tdareasons (
 );
 
 -- Table comment
-COMMENT ON TABLE tdareasons IS '{"label": "Reasons for governance request", "description": "Outcomes set by the Technical Design Authority", "schemalastupdated": "06/03/2019", "dataversion": 1}';
+COMMENT ON TABLE tdareasons IS '{"label": "Reasons for governance request", "description": "Outcomes set by the Technical Design Authority.", "schemalastupdated": "06/03/2019", "dataversion": 1}';
```
Comment (T0MT33) [inline schemas/reference/V34__tdareasons.sql]: Further discussion with Tom, a better description would now be "Reasons for attending a TDA"

File: schemas/reference/V31__modetypes.sql [hunk 2]
```diff
@@ -6,12 +6,12 @@ CREATE TABLE modetypes (
 );
 
 -- Table comment
-COMMENT ON TABLE modetypes IS '{"label": "Class of trasnport", "description": "A list of transport types", "schemalastupdated": "06/03/2019", "dataversion": 1}';
+COMMENT ON TABLE modetypes IS '{"label": "Class of trasnport", "description": "A list of transport types.", "schemalastupdated": "06/03/2019", "dataversion": 1}';
```
[the remark this answers is not in the document]
Comment (JimmySunDance) [reply schemas/reference/V31__modetypes.sql]: Fixed

File: schemas/reference/V34__tdareasons.sql (hunk 1 above)
Comment (JimmySunDance) [reply schemas/reference/V34__tdareasons.sql]: Fixed

File: schemas/reference/V33__tdaapprovalstatus.sql [hunk 3]
```diff
@@ -6,12 +6,12 @@ CREATE TABLE tdaapprovalstatus (
 );
 
 -- Table comment
-COMMENT ON TABLE tdaapprovalstatus IS '{"label": "Governance approval", "description": "A list detailing governance approval status", "schemalastupdated": "06/03/2019", "dataversion": 1}';
+COMMENT ON TABLE tdaapprovalstatus IS '{"label": "Governance approval", "description": "A list detailing governance approval status.", "schemalastupdated": "06/03/2019", "dataversion": 1}';
```
[the remark this answers is not in the document]
Comment (JimmySunDance) [reply schemas/reference/V33__tdaapprovalstatus.sql]: Fixed

**Pull request merged**